core/chain-configs/src/client_config.rs

```diff
@@ -384,6 +389,7 @@ impl ClientConfig {
                 "state_split_config",
             ),
             tx_routing_height_horizon: 4,
+            produce_chunk_add_transactions_time_limit: Duration::from_millis(200),
```

Comment:
It's a config for test purposes and all other values are also specified by hand.
For example, this config has:
```rust
header_sync_stall_ban_timeout: Duration::from_secs(30),
```

Even though the default value is different:
```rust
fn default_header_sync_stall_ban_timeout() -> Duration {
    Duration::from_secs(120)
}
```

I agree that it would be nicer to use the real default value instead of handwritten ones. I will modify it to use the default value function.
The only downside is that it will introduce a cyclical dependency (`Config` <-> `ClientConfig`). Some people dislike that, but that feels very purist, I think it's okay in this case.

Reply:
I guess we could move the default value functions to `near-chain-configs` (?)

Reply:
Moved the default values for ClientConfig to the same module as ClientConfig, this makes it possible to use the default value in `ClientConfig::test` without any circular dependencies